dallinger/command_line.py

```diff
@@ -277,16 +286,13 @@ def debug(verbose, bot):
     logfile = config.get('logfile')
     if logfile != '-':
         logfile = os.path.join(cwd, logfile)
-    config.extend({
-        "mode": u"debug",
-        "loglevel": 0,
-        "logfile": logfile
-    })
-    config.write()
+
+    # Drop all the tables from the database.
+    db.init_db(drop_all=True)
 
     # Start up the local server
     log("Starting up the server...")
-    port = config.get('port')
+    port = config.get('port', 22362)
```

Comment:
This was left in after working out some test issues I was having that erroneously made me think it was needed. Will remove it.